index.js

```diff
@@ -5,22 +5,27 @@
 const minimist = require('minimist')
 const fs = require('fs')
 const path = require('path')
-const concat = require('concat-stream')
 const buildReport = require('./template')
 const help = fs.readFileSync(path.join(__dirname, 'help.txt'), 'utf8')
+const ndjson = require('ndjson')
 
 function start () {
   const argv = minimist(process.argv.slice(2), {
     boolean: ['help'],
     alias: {
       input: 'i',
       version: 'v',
-      help: 'h'
+      help: 'h',
+      compare: 'c'
     },
     default: {
     }
   })
 
+  if (argv.compare) {
+    argv._.push(argv.compare)
```

Comment:
you should check if argv._ exists before pushing to it, `if (!argv._) argv._ = []`